content/applications/inventory_and_mrp/manufacturing/management/bill_configuration.rst

```diff
@@ -33,6 +33,10 @@ type, which is *Manufacture this Product*.
 .. image:: media/bills_of_materials_01.png
     :align: center
 
+.. warning::
+   The destination location should **not** be a scrap location.
+   A scrap location is where you put products that you don't need. 
```

Comment:
```suggestion
   The destination location should **not** be a scrap location. A scrap location is where you put
   products that you don't need. 
```
It's best to avoid line returns in RST when a line return is not expected in HTML as contributors regularly confuse the two. If a line return in HTML *is* intended, the RST should have a blank line between the two sentences.